cli/command/system/info.go

```diff
@@ -91,8 +90,7 @@ func prettyPrintInfo(dockerCli command.Cli, info info) error {
 	}
 
 	fmt.Fprintln(dockerCli.Out())
-	fmt.Fprintln(dockerCli.Out(), "Daemon")
-	fmt.Fprintln(dockerCli.Out(), "------")
+	fmt.Fprintln(dockerCli.Out(), "Daemon:")
```

Comment:
😅 erm; should this be `Server:` (to be consistent with `docker version`)?

```suggestion
	fmt.Fprintln(dockerCli.Out(), "Server:")
```

Reply:
Will do. Luckily they are the same length so it's easy(ish) to push through the series without worrying about alignment issues etc ;-)